apps/studio/src/lib/db/clients/sqlserver.js

```diff
@@ -37,8 +37,11 @@ export default async function (server, database) {
   // light solution to test connection with with the server
   await driverExecuteQuery(conn, { query: 'SELECT 1' });
 
+  const version = await getVersion(conn);
```

Comment:
You can probably delete the previous line (SELECT 1) as getting the version will test the connection also